Main.tscn

```diff
@@ -0,0 +1,29 @@
+[gd_scene load_steps=3 format=2]
```

Comment:
Don't include test files with your commit, only keep changes to the plugin itelf.